engine-rest/engine-rest-openapi/src/main/templates/paths/external-task/fetchAndLock/post.ftl

```diff
@@ -67,7 +67,7 @@
                          "workerId": "aWorkerId",
                          "maxTasks": 2,
                          "usePriority": true,
-                         "sorting": [
+                         "sortings": [
```

Comment:
❌ Wouldn't changing this property to `sorting` be more consistent? I think most of the other REST API properties for sorting are called `sorting`? Or am I missing something?

Reply:
FYI @danielkelemen, changing the name would affect https://github.com/camunda/camunda-bpm-platform/issues/3928.

Reply:
@tasso94 Doesn't the OpenAPI example4 need to reflect the request sent to the `/fetchAndLock ` POST operation? Currently the request DTO that is used has a field called `sortings`. I understood the whole point of the ticket was to align the examples with the existing parameter.